default.nix

```diff
@@ -180,13 +176,6 @@ let
       };
     };
 
-    walletIntegrationTests = self.callPackage ./scripts/test/wallet/integration {
-      inherit (self.cardanoPackages)
-        cardano-sl-tools
-        cardano-wallet;
-      inherit useStackBinaries;
-    };
-
```

Comment:
Nix scripts is now completely outdated. Integration tests are ran directly from stack.